articles/services/performance-scalability.md

```diff
@@ -0,0 +1,25 @@
+---
+description: Project to assist customers with common performance testing needs, including a correct planning, and test automation guidance.
+---
+
+# Geo High Availability
```

Comment:
@eugeniop There are 2 "Geo High Availability" items, looks like this one is wrong.

https://auth0-docs-content-pr-2798.herokuapp.com/docs/services > Assess & Plan

![image](https://cloud.githubusercontent.com/assets/11715799/20433188/3b88b11c-adab-11e6-9f92-4e00cec5f402.png)